python/src/wslink/protocol.py

```diff
@@ -252,7 +254,7 @@ async def onMessage(self, is_binary, msg, client_id):
             payload = payload.decode("utf-8")
 
         rpc = json.loads(payload)
-        logging.debug("wslink incoming msg %s" % self.payloadWithSecretStripped(rpc))
+        logger.debug("wslink incoming msg %s" % self.payloadWithSecretStripped(rpc))
```

Comment:
We may want to fix that line (while we are at it) to prevent string construction when the logger is disabled.

Reply:
Could you fix that with the following?

`logger.debug("wslink incoming msg %s", self.payloadWithSecretStripped(rpc))`

Reply:
```suggestion
        logger.debug("wslink incoming msg %s", self.payloadWithSecretStripped(rpc))
```